.prettierrc

```diff
@@ -1,6 +1,6 @@
 {
   "bracketSpacing": true,
-  "printWidth": 150,
+  "printWidth": 100,
```

Comment:
Should this be `120` too?

Reply:
As in - there is a conflict here with `max-len` from `.eslintrc` which you have set to 120.

Which is it - 100 or 120?

Reply:
These rules have different application actually - prettier applies to the code that's being generated from the AST, which ignores certain things like strings and comments. Explanation here:
https://github.com/prettier/eslint-config-prettier#max-len

So 100 is the soft target applied to code generated by prettier, while 120 is the hard limit enforced by eslint.